gatsby/sourceNodes.ts

```diff
@@ -75,6 +75,91 @@ export const sourceNodes: GatsbyNode['sourceNodes'] = async ({ actions, createCo
         components: JSON.stringify(spec.components),
     })
 
+    // PostHog SDK references
+
+    // Paths to the SDK references folders
+    const SDK_REFERENCES_FOLDER_PATHS = [
+        {
+            id: 'posthog-python',
+            repo: 'gewenyu99/posthog-python',
```

Comment:
I will update these to point to real SDK repo folders once I merge changes there